src/mlpack/methods/cf/cf_model_impl.hpp

```diff
@@ -105,15 +118,49 @@ void CFModel::Train(const MatType& data,
                     const size_t rank,
                     const size_t maxIterations,
                     const double minResidue,
-                    const bool mit)
+                    const bool mit,
+                    const std::string& normalization)
 {
   // Delete the current CFType object, if there is one.
   boost::apply_visitor(DeleteVisitor(), cf);
 
   // Instantiate a new CFType object.
   DecompositionPolicy decomposition;
-  cf = new CFType<DecompositionPolicy>(data, decomposition,
-      numUsersForSimilarity, rank, maxIterations, minResidue, mit);
+  if (normalization == "overallMean")
+  {
+    cf = new CFType<DecompositionPolicy, OverallMeanNormalization>(data,
+        decomposition, numUsersForSimilarity, rank, maxIterations, minResidue,
+        mit);
+  }
+  else if (normalization == "itemMean")
```

Comment:
I'd suggest using underscores here---it will match all of the other string options in all of the other bindings.  So, e.g., `item_mean` and `user_mean`.

Reply:
I have updated. :+1: